scripts/mkcloud

```diff
@@ -115,6 +117,9 @@ fi
 # causes of this is violation of the common shell coding standard which
 # uses uppercase for environment variables and constants, and lowercase
 # for local variables.
+: ${me:=$0}
+git_qa_crowbarsetup=$(dirname $me)/qa_crowbarsetup.sh
+: ${qa_crowbarsetup:=$git_qa_crowbarsetup}
```

Comment:
could be a one-liner (IMHO as readable):
: ${qa_crowbarsetup:=$(dirname $0)/qa_crowbarsetup.sh}

or at most do script_dir=$(dirname $0)